doc/operations/platforms/snp.rst

```diff
@@ -18,13 +18,14 @@ Confidential Azure Container Instance (ACI)
 
 .. note:: See `here <https://learn.microsoft.com/en-us/azure/container-instances/container-instances-tutorial-deploy-confidential-containers-cce-arm>`_ for more information on the deployment of confidential containers in Azure.
 
-Azure Confidential ACI provides a security context directory containing the following files. The content of these files are checked against the attestation report on node startup and join, and stored in the ledger for audit and improved serviceability. 
+Azure Confidential ACI provides a security context directory containing the following files.
 
-- ``host-amd-cert-base64``: The certificate chain corresponding to the key (VCEK) used to sign the attestation report, up to the well-known AMD root of trust certificate authority (Base64 encoded). 
```

Comment:
This is still provided, but subject to a race condition on startup if PSP upgrades take place at the wrong time. Resolution through an online service such as THIM using the chip id should be used instead.